compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/platform/WebTextInputService.kt

```diff
@@ -100,8 +100,12 @@ internal abstract class WebTextInputService : PlatformTextInputService, InputAwa
         decorationBoxBounds: Rect
     ) {
         val newRect = getNewGeometryForBackingInput(innerTextFieldBounds)
```

Comment:
`innerTextFieldBounds` and `FocusedRect` are different rectangles.
if `updateTextLayoutResult` gets called sometine later, it will conflict with `notifyFocusedRect`.

I'd keep only `notifyFocusedRect`.